lib/model/content.dart

```diff
@@ -297,7 +297,7 @@ class CodeBlockNode extends BlockContentNode {
   }
 }
 
-class CodeBlockSpanNode extends InlineContentNode {
+class CodeBlockSpanNode extends ContentNode {
```

Comment:
```
content [nfc]: Make CodeBlockSpanNode inherit ContentNode.

We only expect CodeBlockSpanNode within a CodeBlockNode, and the
InlineContent widget should not be responsible for building it, as the
widget does for all other InlineContentNode.

Signed-off-by: Zixuan James Li <zixuan@zulip.com>
```

Commit message nit: "Make CodeBlockSpanNode inherit ContentNode" seems accidentally misleading, because it already was inheriting from `ContentNode`, just more distantly. (`InlineContentNode` extends `ContentNode`.)

How about something like "Make CodeBlockSpanNode extend ContentNode directly"?

Reply:
Thanks! Fixed in the latest push.